packages/segmented-control/src/SegmentedControl.tsx

```diff
@@ -295,8 +295,12 @@ const SegmentedControl = React.forwardRef<
   }: SegmentedControlProps,
   forwardedRef,
 ) {
-  // TODO log warning if defaultValue is set but does not match any child value
+  // TODO: log warning if defaultValue is set but does not match any child value
   const { usingKeyboard } = useUsingKeyboardContext();
+  const [segmentedContainer, setSegmentedContainer] = useState<null | Element>(
```

Comment:
🤷 Why use this pattern, vs `useRef`?

Reply:
Good question! Turns out `useRef` works fine.